packages/core/src/app/giftCertificate/AppliedGiftCertificate.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { getStoreConfig } from '@bigcommerce/checkout/test-mocks'
+import { render, screen } from '@bigcommerce/checkout/test-utils';
+
+import { default as AppliedGiftCertificate } from './AppliedGiftCertificate';
+import { getGiftCertificate } from './giftCertificate.mock';
+
+function roundTo(amount: number, decimals: number): string {
+    const factor = 10 ** decimals;
+
+    return (Math.round(amount * factor) / factor).toFixed(decimals);
+}
+
+describe('AppliedGiftCertificate', () => {
+    const config = getStoreConfig();
+    const giftCertificate = getGiftCertificate();
+
+    it('renders correctly if gift certificate is applied', () => {
+        render(<AppliedGiftCertificate giftCertificate={giftCertificate} />);
+
+        const usedAmount = roundTo(giftCertificate.used * config.shopperCurrency.exchangeRate, parseInt(config.shopperCurrency.decimalPlaces, 10));
+        const remainingAmount = roundTo(giftCertificate.remaining * config.shopperCurrency.exchangeRate, parseInt(config.shopperCurrency.decimalPlaces, 10));
+
+        const usedAmountElement = screen.getByTestId('giftCertificate-amount');
+        const remainingAmountElement = screen.getByTestId('giftCertificate-remaining');
```

Comment:
Have to use this since currency formatter is an internal implementation in SDK, so it's ok to check for text content which is the amount minus the currency symbol.